app/models/type_de_champ.rb

```diff
@@ -578,6 +589,24 @@ def routable?
     type_champ.in?(ROUTABLE_TYPES)
   end
 
+  def invalid_regexp?
+    return false if expression_reguliere.blank?
+    return false if expression_reguliere_exemple_text.blank?
+    begin
+      if expression_reguliere_exemple_text.match?(Regexp.new(expression_reguliere, timeout: 2.0))
+        return false
+      end
+    rescue Timeout::Error
```

Comment:
dans ce begin, on veut le Regexp:TimeoutError
```suggestion
    rescue Regexp::TimeoutError 
```

je pense qu'on peut mm enlever le begin/rescue/end pour un simple rescue comme  `rescue RegexpError`